src/state/workspaces/selectors.ts

```diff
@@ -0,0 +1,10 @@
+import { createSelector } from '@reduxjs/toolkit';
+import { selectRootState } from '../store';
+
+export const selectWorkspacesState = createSelector(selectRootState, (state) => state.workspaces);
+
+export const selectWorkspacesList = createSelector(selectWorkspacesState, (state) =>
```

Comment:
I'm considering just inserting workspaces already sorted, but for now it can stay in the selector. No issue with moving it out, though, especially since we have to spread the list to sort it appropriately.

Reply:
Sorting here shouldnt be very expensive as realistically we're dealing with < 20 max